### PR TITLE
Highlight dropped element automatically

### LINKED ISSUE #783: Highlight element automatically when dropped onto the grid
When an element is dragged onto the grid, it should be highlighted by default.

Acceptance criteria:
* Highlighted by default without having to select it manually.
* The crown config should be open.
* The inspector panel should have the element's config loaded.
* Clicking away should de-select the element as normal.

### PR DESCRIPTION
**Pull request by chipit24**
Fixes #783.

### CODE REVIEW

File: src/mixins/crownConfig.js [hunk 1]
```diff
@@ -188,6 +191,12 @@ export default {
       });
     },
     configureCrown() {
+      if (this.crownConfigured) {
+        return;
+      }
+
+      this.crownConfigured = true;
+
```
Comment (JCook21) [inline src/mixins/crownConfig.js]: This seems a little weird. What do these lines achieve and why are they needed?

File: tests/unit/nodeManager.spec.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
+import { addIdToNodeAndSetUpDiagramReference, addNodeToProcess, getTargetProcess } from '@/components/nodeManager';
```
Comment (JCook21) [inline tests/unit/nodeManager.spec.js]: I really like that you've done the code extraction and added unit tests! Great job. 💯

File: src/mixins/crownConfig.js (hunk 1 above)
Comment (chipit24) [reply src/mixins/crownConfig.js]: It's to prevent this function from running twice. Running it twice duplicates the crown buttons. Without highlight on drop, this method was only called once when the element was clicked. But with the crown already visible on drop, I don't want this function called again if a user clicks the element.

I'll see if I can clean this up.

Comment (chipit24) [reply src/mixins/crownConfig.js]: I removed `crownConfigured` and made a single call to `configureCrown` within `setUpCrownConfig`. This may have some performance implications (as the call to `configureCrown` was deferred to help with performance on initial load), but as long as tests continue to pass, performance issues will just have to be re-assessed in the future; I think this is a fair trade-off.

File: package.json [hunk 3]
```diff
@@ -118,11 +118,10 @@
   ],
   "nyc": {
     "extension": [
-      ".js",
       ".vue"
     ],
-    "exclude": [
-      "tests/**/*.js"
+    "include": [
+      "src/**"
```
Comment (JCook21) [inline package.json]: Why is this change needed?

**Comment by chipit24** [comment]
FYI: I force pushed to remove the test commits I pushed up and to pull in the latest changes. The code should be the exact same as when initially reviewed.

**Comment by chipit24** [comment]
Added WIP label as I work on getting this feature to work in Safari.

**Comment by chipit24** [comment]
Tested in processmaker core on macOS in Chrome and Safari, and in Edge on Windows.
